pyfolio/plotting.py

```diff
@@ -610,16 +629,19 @@ def plot_rolling_returns(
             lw=4, color='red', alpha=0.6,
             label='Live', ax=ax, **kwargs)
 
-        if cone_std is not None:
-            cone_df = timeseries.cone_rolling(
-                returns,
-                num_stdev=cone_std,
-                cone_fit_end_date=live_start_date)
-
-            cone_df_fit = cone_df[cone_df.index < live_start_date]
+        if cone_std is not None and not volatility_match:
```

Comment:
I think we shouldn't prohibit a user from drawing a cone if they set volatility_match to true. Instead, we should pass `cone_std=None` in the `create_returns_tearsheet()` function.